### DISCUSSION TITLE
Support with free licences for tech communities.

### DISCUSSION THREAD

**Discussion by mateuszbialowas** [community/community#27313] [Copilot Conversations]
Hi,
do you know if GitHub supports local tech communities with free GitHub copilot licenses? I am a co-organizer tech user group and I thought to organize a giveaway on GitHub copilot license. Does GitHub have a similar support program as JetBrains? https://www.jetbrains.com/community/user-groups/

**Comment by derekxmartin** [comment]
I don’t understand how this would work as GPT-3 is still a cost (an expensive one at that). Just my 2c.
